### ISSUE TITLE
Allow overriding of theme variable with explicit styles prop on Tab component

### ISSUE THREAD

**Issue by audiolion** [GeekyAnts/NativeBase#2134]
With 2.6.0 (the next release after v2.5.2) the ability to customize the Tab component font size through the theme (`tabFontSize`) was added. Unfortunately, this style is placed at the end and so overrides explicit `fontSize` styles one would pass to a specific Tab component instance (through `textStyle` and `activeTextStyle`). This specifically occurs when you are using the `<Tab heading="TabName" />` interface and not providing your own component.

The code section where this occurs:

https://github.com/GeekyAnts/NativeBase/blob/0748a749f539f792cc7719ffaf31c603c23f1cad/src/basic/Tabs/DefaultTabBar.js#L61-L77

My proposal would be to put `{ fontSize: tabFontSize }` before the `isTabActive` style computation so it would look like:

```js
<Text
  style={[
    { fontSize: tabFontSize },
    isTabActive ? activeTextStyle : textStyle,
  ]}
>
```

instead and if you explicitly choose to pass `activeTextStyle = { fontSize: 12 }; textStyle = { fontSize: 12 }` to the component it would apply correctly.

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.7.2](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.7.2)

**Comment by audiolion** [comment]
Thanks for the quick response!
